### PR TITLE
PMCP-3704 & PMCP-3777 & PMCP-3776 & PMCP-3778 & PMCP-3780 - RS Header

### PR DESCRIPTION
**Pull request by prajapatisagar**
Link to Header module: https://pmc-larva-git-feature-header-penske-media-corp.vercel.app/larva/__tests__/vlanding/

### Make sure you complete these items:

- [x] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [x] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))
- [ ] If adding a new pattern, in the PR comment, included a screenshot and link to the static Vercel deployment
- [ ] If changes to build scripts or the Node.js server, tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw)
- - If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

File: packages/larva-css/src/03-utilities/u-font-size.common.inline.scss [hunk 1]
```diff
@@ -95,4 +95,10 @@
 	@include pmc-breakpoint( mobile-max ) {
 		font-size: pmc-rem(13);
 	}
+}
+
+.lrv-u-font-size-8\@mobile-max {
```
Comment (laras126) [inline packages/larva-css/src/03-utilities/u-font-size.common.inline.scss:100]: Can this be incorporated into a token for lrv-a-font?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## Unpublished
+* larva-patterns - Add `expandable-search`, `header-menu`, `header-sticky`, `header`, `search-form`, `read-next` modules.
```
Comment (laras126) [inline CHANGELOG.md:7]: Please address the checkboxes in the PR comment.

File: packages/larva-css/src/04-js/partials/expandable-search.scss [hunk 3]
```diff
@@ -0,0 +1,207 @@
+@import 'setup';
+
+/* stylelint-disable */
+// Temporarily disabled until reactor is merged.
```
Comment (laras126) [inline packages/larva-css/src/04-js/partials/expandable-search.scss:4]: What does this mean?

**Review by laras126 — CHANGES_REQUESTED** [review]
Need a couple of updates - particularly to add links to the modules in the PR comment for the reviewers.

**Comment by laras126** [comment]
Hi Sagar - (re-iterating what I wrote in Slack) definitely update the PR with links and screenshots for the added modules, but please wait to remove the very specific CSS values until we have made an official decision about this. Thank you!

**Review by vtellapmc — APPROVED** [review]

**Pull request merged**